Sources/NIO/EventLoop.swift

```diff
@@ -244,9 +244,17 @@ public protocol EventLoop: EventLoopGroup {
     @discardableResult
     func scheduleTask<T>(in: TimeAmount, _ task: @escaping () throws -> T) -> Scheduled<T>
 
-    /// Checks that this call is run from the `EventLoop`. If this is called from within the `EventLoop` this function
-    /// will have no effect, if called from outside the `EventLoop` it will crash the process with a trap.
+    /// Asserts that the current thread is the one tied to this `EventLoop`. Has no effect if `inEventLoop` is `true`.
```

Comment:
This statement is worth removing: while it’s true, it implies that the behaviour of this function is in terms of `inEventLoop`. However, it isn’t.